labrad/server.py

```diff
@@ -353,7 +372,23 @@ def _connectionMade(self, protocol):
         """Login as a server after connecting to LabRAD."""
         try:
             addr = protocol.transport.getPeer()
-            log.msg('connected to %s:%s' % (addr.host, addr.port))
+            log.msg('Connected to {}:{}'.format(addr.host, addr.port))
+            if self._tls_mode == 'starttls' and addr.host != '127.0.0.1':
+                log.msg('Manager is remote ({}). Starting TLS.'.format(addr.host))
+                host = self._remote_host
+                try:
+                    yield protocol.sendRequest(C.MANAGER_ID, [(1L, ('STARTTLS', host))])
+                except Exception:
```

Comment:
It might be useful to also include this Exception's message in the message of the raised Exception - it can help debugging why an exception occurred (particularly since this catching all exceptions).